sensu/standalone_test.go

```diff
@@ -0,0 +1,116 @@
+package sensu
+
+import (
+	"testing"
+
+	stdCheck "github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/check"
+	stdClient "github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/client"
+)
+
+func TestMissingCommandKey(t *testing.T) {
+	standaloneProcessor := &Standalone{check: &stdCheck.Check{}}
+
+	err := standaloneProcessor.execute()
+
+	if err != commandKeyError {
+		t.Fatalf("Expected error to be \"%s\" but got \"%s\" instead!",
```

Comment:
Ah, yes. Silly me! I should have read your comment above more carefully. I am using the `goimports` tool to reformat the code on save and it kept relocating the closing bracket :) I didn't notice the trailing comma trick.